pkg/authorization/authz_unix_test.go

```diff
@@ -172,6 +172,54 @@ func TestDrainBody(t *testing.T) {
 	}
 }
 
+func TestSendBody(t *testing.T) {
+	var (
+		url       = "nothing.com"
+		testcases = []struct {
+			contentType string
+			expected    bool
+		}{
+			{
+				contentType: "application/json",
```

Comment:
Perhaps to be complete, we should have a mixed-case example here as well, e.g.  `Application/JSON`, because they [are case-insensitive](https://www.w3.org/Protocols/rfc1341/4_Content-Type.html) (`mime.ParseMediaType()` already normalises that, so no changes to the code are needed)

Reply:
I added 3 testcases more.